### PR TITLE
Project list test evolution widget

### PR DESCRIPTION
**Pull request by Prevole**
Ready for pre-review.

### CODE REVIEW

**Comment by Prevole** [comment]
Ready for review

File: app/views/templates/project-panel.html.slim [hunk 1]
```diff
@@ -21,7 +21,9 @@
...
           strong Getting Started
         '  guide to set it up.
-    .col-md-3.col-sm-4.col-xs-12 ng-if='stateChart'
+    .col-lg-4.col-md-5.col-sm-12 ng-if='project'
+      test-activity-chart ng-if='currentOrganization && project' type='projects' nb-days='15' organization='currentOrganization' project='project'
+    .col-lg-3.col-md-3.col-sm-12 ng-if='stateChart'
       .latest-state
         p.text-info Results of the last run
-        canvas.chart.chart-pie data='stateChart.data' labels='stateChart.labels' colours='stateChart.colors'
+        canvas.chart.chart-pie height='200px' data='stateChart.data' labels='stateChart.labels' colours='stateChart.colors'
```
Comment (AlphaHydrae) [inline app/views/templates/project-panel.html.slim]: The browser is nice, but `height='200px'` is actually incorrect. The height attribute of a canvas is in pixels by definition, so it should be `height='200'`.

File: features/new_tests_by_day_metrics.feature [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+        "date": "@date(1 day ago)",
+        "testsCount": 1
+      }, {
+        "date": "@date(today)",
+        "testsCount": 0
+      }]
+      """
+    And nothing should have been added or deleted
+
+
```
Comment (AlphaHydrae) [inline features/new_tests_by_day_metrics.feature]: Please add one successful test for an anonymous user and a public organization.

File: features/step_definitions/test_steps.rb [hunk 3]
```diff
@@ -6,12 +6,19 @@
...
+  end
+
   options = {
     name: name,
     key: key,
     project: project,
     last_runner: user,
-    project_version: project_version
+    project_version: project_version,
+    first_run_at: date
```
Comment (AlphaHydrae) [inline features/step_definitions/test_steps.rb]: Also set `created_at` for consistency.

File: app/views/templates/project-panel.html.slim (hunk 1 above)
Comment (Prevole) [inline app/views/templates/project-panel.html.slim]: Fixed

File: features/new_tests_by_day_metrics.feature (hunk 2 above)
Comment (Prevole) [inline features/new_tests_by_day_metrics.feature]: Fixed

**Pull request merged**